ush/parsing_namelists_DATM.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Use `#! /usr/bin/env bash` to be consistent with other scripts.

Reply:
Done